controller_manager/src/controller_manager.cpp

```diff
@@ -188,6 +201,26 @@ ControllerManager::ControllerManager(
   init_services();
 }
 
+void ControllerManager::init_resource_manager_cb(const std_msgs::msg::String & robot_description)
+{
+  RCLCPP_ERROR(
+    get_logger(), "'init_resource_manager_cb called with %s", robot_description.data.c_str());
+  // TODO(Manuel) errors should probably be caught since we don't want controller_manager node
```

Comment:
The error should be managed by parser, or resource manager. CM doesn't do much anything with robot description.